app/src/main/res/layout/fragment_recent_transactions.xml

```diff
@@ -4,11 +4,41 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <android.support.v7.widget.RecyclerView
-        android:id="@+id/rv_recent_transactions"
-        android:layout_width="wrap_content"
-        android:layout_height="0dp"
-        android:layout_centerHorizontal="true"
-        android:layout_weight="1"/>
+    <LinearLayout
```

Comment:
There no hierarchy of using xml view. bad practice.
Please look this as an example : https://github.com/openMF/android-client/blob/master/mifosng-android/src/main/res/layout/fragment_client.xml